src/adv.py

```diff
@@ -39,6 +40,29 @@
 
 # Make a new player object that is currently in the 'outside' room.
 
+
+exited = False
+
+
+player = Player('outside')
```

Comment:
might be missing an arg here?

Reply:
Discussed that a second arg isn't needed because it was removed from the player init 👍